helm/reana/Chart.yaml

```diff
@@ -31,7 +31,7 @@ version: 0.9.0
 kubeVersion: ">= 1.19.0-0 < 1.26.0-0"
 dependencies:
   - name: traefik
-    version: 10.3.4
+    version: 10.6.2
```

Comment:
Updated to the same version used by the latest `kubernetes-1.25.3-3` CERN k8s template